configs/components/jsbach/jsbach.yaml

```diff
@@ -103,8 +103,8 @@ choose_version:
 
                         ssp534os:
                                 scenario_type: cmip6
-
-                        cmip6hist:
+                        
```

Comment:
```suggestion

```

Reply:
Talking about ocd... I have vim configure to show trailing spaces in red, so it kind of hurts on the eyes